src/app/PCM/templates/tpl-detail-view/tpl-property-detail-view/index.js

```diff
@@ -78,9 +83,23 @@ const TemplatePropertyDetailView = ({ mappedPcm }) => {
 
   //PRI_IMAGES is now used for the multi image display
   const imageCode = findCode('PRI_IMAGES') || findCode('PRI_IMAGE_URL')
+
   //Get suburb, state if both not null, otherwise just the one that isn't null
-  const location =
-    !!suburb && !!state ? `${suburb}, ${state}` : !!addressFull ? addressFull : suburb || state
+  const getLocation = () => {
+    let _suburb = suburb || addressSuburb
+    let _state = state || addressState
+    if (doubleBang(_suburb) && doubleBang(_state)) {
+      return `${suburb}, ${state}`
+    }
```

Comment:
don't need to convert it in to a function as we aren't taking any input based on which we are doing some operations inside the function. location can and is better calculated in the global scope itself, rather than by invoking a function.